src/Jmikola/AutoLogin/Event/AlreadyAuthenticatedEvent.php

```diff
@@ -2,9 +2,7 @@
 
 namespace Jmikola\AutoLogin\Event;
 
-use Symfony\Component\EventDispatcher\Event;
-
-class AlreadyAuthenticatedEvent extends Event
+class AlreadyAuthenticatedEvent
```

Comment:
Per [this deprecation notice](https://github.com/symfony/symfony/blob/v4.3.4/src/Symfony/Component/EventDispatcher/Event.php#L15) in Symfony 4.3.4, would it be preferable to extend [`Symfony\Contracts\EventDispatcher\Event`](https://github.com/symfony/symfony/blob/v4.3.4/src/Symfony/Contracts/EventDispatcher/Event.php) instead?

Reply:
yeah, this is an oversight on my end, the base event is not deprecated but only the namespace has changed. will update.